go.mod

```diff
@@ -26,15 +26,20 @@ require (
 require (
 	github.com/cenkalti/backoff/v4 v4.2.0 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/fatih/color v1.15.0
```

Comment:
We are having direct imports in the first `require`, and indirect in second `require`

Reply:
wonder why they were placed there